src/Game.java

```diff
@@ -0,0 +1,160 @@
+import jodd.json.JsonParser;
+import jodd.json.JsonSerializer;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Scanner;
+
+/**
+ * Created by Nigel on 6/21/16.
+ */
+public class Game {
+    File file = new File("players.json");
+    private int totalSticks;
+    private Player player1, player2, currentPlayer = null;
+    Scanner scanner = new Scanner(System.in);
+    public static final int STICKMAX = 100;
+    public static final int STICKMIN = 10;
+    public static final int SELECTMIN = 1;
+    public static final int SELECTMAX = 3;
+    String winnerName;
+    String loserName;
+
+    public Game(Player player1, Player player2) {
+        this.player1 = currentPlayer = player1;
+        this.player2 = player2;
+    }
+
+    public String setWinner(Player name) {
+        return winnerName = name.getName();
+    }
+
+    public String setLoser(Player name) {
+        return loserName = name.getName();
+    }
+
+    public String getWinner() {
+        return winnerName;
+    }
+
+    public String getLoser() {
+        return loserName;
+    }
+
+    public void runGame() {
+        int choice;
+        setBoard();
+        System.out.println("Ok, let's play!");
+        while (!declareLoser(totalSticks)) {
+            choice = currentPlayer.makeChoice();
+            checkChoice(choice);
+            totalSticks -= choice;
+            System.out.format("There are %d sticks remaining.\n", totalSticks);
+            switchPlayer();
+        }
+    }
+
+    public void setBoard() {
+        int stickSelect = 0;
+        while (stickSelect < STICKMIN || stickSelect > STICKMAX) {
+            System.out.println("How many sticks will be on the table (10 - 100): ");
+            stickSelect = Integer.parseInt(scanner.nextLine());
+            if (stickSelect < STICKMIN || stickSelect > STICKMAX) {
+                System.out.println("Nope, enter a number between 1 & 100.");
+            }
+        }
+        totalSticks = stickSelect;
+    }
+
+    public void switchPlayer() {
+        if (currentPlayer == player1) {
+            currentPlayer = player2;
+        } else {
+            currentPlayer = player1;
+        }
+    }
+
+    public boolean declareLoser(int amountRemaining) {
```

Comment:
The code in the 2 if statements is almost entirely duplicated.  

`if(amountRemaining == 1){
    switchPlayer();
}`

Would effectively eliminate 6 lines of code.